prettyPrinters/rust_types.py

```diff
@@ -35,7 +35,7 @@ class RustType(object):
 
 # Should be synchronized with `RsDebugProcessConfigurationHelper.RUST_STD_TYPES`
 STD_STRING_REGEX = re.compile(r"^(alloc::([a-z_]+::)+)String$")
-STD_STR_REGEX = re.compile(r"^&str$")
+STD_STR_REGEX = re.compile(r"^[&*]?(const |mut )?str[*]?$")
```

Comment:
Let's add a comment on why we need so complex (and incorrect in rust type terms) regex

Reply:
I've already described this in the commit message body. The list of supported Rust types is quite big so I decided not to repeat it in a comment.

Do you think a commit message is not enough in such cases? I'd like to get your opinion on it because sometimes I use commit message bodies for descriptions but now I'm not pretty sure if they are convenient for others.

Reply:
From my point of view, commit message is less convenient for reader because
- with comment, you don't need to do anything else, just read the code
- if someone makes a change on this line, it will take more time to find the commit that introduced necessary change to read its message

At the same time, comments tend to be outdated so I'd prefer to have both comment to simplify reading in most cases and useful commit message to get the truth if something happens with comment

Reply:
Thank you for your suggestion, I added a comment. I will try to use both commit messages and comments for such explanations